packages/top-nav/src/top-nav-item.css

```diff
@@ -45,11 +45,11 @@ a:focus-visible {
         var(--mod-tabs-top-to-text, var(--spectrum-tabs-top-to-text)) / 2
     );
     inset-inline: calc(
-            var(
-                    --mod-tabs-focus-indicator-gap,
-                    var(--spectrum-tabs-focus-indicator-gap)
-                ) * -1
-        );
+        var(
```

Comment:
This shouldn't be a part of this PR. Can you make sure this PR is clean with only related files